stix_shifter_modules/ibm_security_verify/stix_translation/json/to_stix_map.json

```diff
@@ -116,6 +116,30 @@
       "key": "x-oca-event.created",
       "transformer": "EpochToTimestamp",
       "object": "ocaevent"
+    },
+    {

```

Comment:
why do you need `time` field mapped to first+last observed when you already mapped them to startime and endtime?